pages/homePage.js

```diff
@@ -11,6 +11,16 @@ const NAVIGATION_BAR_TAB = {
   NEW_YEARS_TREES: "Новогодние елки",
 };
 
+async function clickSelectorHeadPage(selectorName) {
+  const headPageSelector = `//span[contains(@class, "navigation__text") and text() = "${selectorName}"]`;
+  await element(headPageSelector).clickElement();
+}
+
+async function clickSelectorCatalogBar(nameBar) {
+  const selectorBar = `//span[contains(@class, "catalog-navigation-classifier__item-title-wrapper") and text() = "${nameBar}"]`;
+  await element(selectorBar).clickElement();
+}
+
```

Comment:
It is not clear what are these functions for
Please provide more appropriate naming